inbm/dispatcher-agent/dispatcher/remediationmanager/remediation_manager.py

```diff
@@ -138,21 +130,15 @@ def _remove_container(self, ids: Any) -> None:
                 if err:
                     logger.error("Error encountered while getting container ID")
 
-                if not temp_image_name in str(active_containers_list) or "DBS" in container_id:
-                    self._dispatcher_callbacks.broker_core.telemetry(
-                        'DBS Security issue raised on containerID: ' +
-                        str(container_id) + ' not present in list.')
-                    continue
-
                 if temp_image_name in str(active_containers_list) and not self.dbs_remove_image_on_failed_container:
                     self.container_image_list_to_be_removed.append(temp_image_name)
 
                 if self.dbs_remove_image_on_failed_container:
                     image_id, image_name = self._get_image_id(trtl, container_id)
                     if image_id is None:
                         raise ValueError('Cannot read image ID')
-
-                (out, err, code) = trtl.stop_by_id(str(container_id))
+                logger.debug(container_id)
```

Comment:
What is the difference between stop_all and stop_all_by_id?

Reply:
If any Container/Image has been failed, before going to removal, we should stop the container. 
stop_all Container to be stopped by parsing container Name as a parameter.
stop_by_id stopping container by ID but parsing Name as a parameter instead of Id, so that unable to stop the failed containers.